lib/smart_answer_flows/pay-leave-for-parents/questions/_warning_hint.govspeak.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :body do %>
+  %Answer these questions about one job only. You'll need to use the calculator again if you have another job.%
+<% end %>
```

Comment:
I think what you had before, a content `hint` was better. You just needed to remove the callout tags.

A body tag with a callout makes all the text on the page large and bold: https://smart-answers-pr-2811.herokuapp.com/pay-leave-for-parents/y/yes/2016-12-31/employee/employee

I would run it past a content designer to make sure that was what they intended.